src/models/Event.js

```diff
@@ -0,0 +1,53 @@
+const Group = require("./Group");
+const mongoose = require("mongoose");
+
+const eventSchema = new mongoose.Schema(
+  {
+    groupID: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "Group",
+    },
+    name: {
+      type: String,
+      required: [true, "please provide an activity name!"],
+    },
+    description: {
+      type: String,
+      default: "a default event description",
+    },
+    eventDateTime: {
+      type: Date,
+    },
+    host: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: "User",
+    },
+    participants: [
```

Comment:
Maybe you all discussed this in a meeting recently...are an event's participants different than the group members? Do group members explicitly join an event or are they all event participants automatically?

Reply:
Yes. My thinking behind having a `participants` field is that not all group members will vote for the same activity to be selected as the 'chosen' activity on an event. So the participants field will only be filled with those group members that have voted for the 'chosen' activity. And as for as joining an event is concerned, I am currently in the process of writing the logic for that, and so far I am thinking that group members will only be part of an event when they have voted for the 'chosen' activity (activity with majority votes).

Reply:
Yes that is correct. I'll consider this change on my next PR related to Chosen Activity [issue](https://github.com/Code-the-Dream-School/ee-prac-team2-back/issues/50).